internal/configmap/mock_configmap.go

```diff
@@ -12,9 +12,9 @@ import (
 	context "context"
 	reflect "reflect"
 
-	v10 "github.com/openshift/cluster-nfd-operator/api/v1"
+	v1 "github.com/openshift/cluster-nfd-operator/api/v1"
 	gomock "go.uber.org/mock/gomock"
-	v1 "k8s.io/api/core/v1"
+	v10 "k8s.io/api/core/v1"
```

Comment:
are we just swapping the v1 and v10 labels around here?  it doesn't look like it should make any difference so I'm not sure I understand why we're doing it.

Reply:
This is the result of running generate command...